Dockerfile.tpl

```diff
@@ -3,11 +3,15 @@ MAINTAINER source{d}
 
 RUN apk add --no-cache --update python python3 py-pip py2-pip git
 
-RUN pip2 install pydetector-bblfsh==0.10.3
 
 ADD build /opt/driver/bin
 ADD native/python_package /tmp/python_driver
 RUN pip3 install /tmp/python_driver
 RUN yes|rm -rf /tmp/python_driver
 
+ADD native/dev_deps /tmp/dev_deps
```

Comment:
would it be possible to have just a default that could be overriden, for example with an env var or similar instead of hardcoding it?